setup.py

```diff
@@ -24,4 +24,5 @@
     python_requires=">=3.10",
     install_requires=requirements,
     zip_safe=False,
+    package_data={"rat": ["grammar.ebnf"]},
```

Comment:
This file wasn't being added when you installed the package, so I've manually stated the file to be included during installation.

Reply:
Oh cool. Did this lead to an error? I do `pip install -e .` to develop locally so I miss stuff like this.

Reply:
I also have it installed in editable mode, but then realized tests ran on actions were giving that filenotfounderror. So I tried installing rat on a brand new venv without the editable flag, and saw that the ebnf file doesn't get copied over.